### PR TITLE
fixes #12712 add search-link for Users and Usergroups in all languages

### PR DESCRIPTION
**Pull request by Apikas1**
fixes #12712 add search-link for Users and Usergroups in all languages

### CODE REVIEW

File: app/helpers/hosts_helper.rb [hunk 1]
```diff
@@ -301,8 +301,8 @@ def overview_fields(host)
     if SETTINGS[:login]
       if host.owner_type == _("User")
```
Comment (domcleal) [inline app/helpers/hosts_helper.rb:302]: I think this needed changing to `== "User"` to fix it?

File: app/helpers/hosts_helper.rb [hunk 2]
```diff
@@ -301,8 +301,8 @@ def overview_fields(host)
...
         fields += [[_("Owner"), (link_to(host.owner, hosts_path(:search => "user.login = #{host.owner.login}")) if host.owner)]]
-      else
-        fields += [[_("Owner"), host.owner]]
+      elsif host.owner_type == ("Usergroup")
```
Comment (domcleal) [inline app/helpers/hosts_helper.rb:304]: The `()` brackets are unnecessary here.

**Comment by domcleal** [comment]
Thanks, I added some inline comments.  You can make the changes and [squash the commits](http://mattsnider.com/amending-and-squashing-commits-in-git/), then push -f again to update this pull request.

If you can also fix the commit message formatting as per the automatic comment, it's missing a hyphen or colon between the ticket number and message.  Use `git commit --amend` for that.

File: app/helpers/hosts_helper.rb (hunk 1 above)
Comment (timogoebel) [inline app/helpers/hosts_helper.rb:302]: .is_a?(User) should also work...

**Pull request closed without merging**